main/acle.rst

```diff
@@ -5807,3 +5820,183 @@ Instructions
 +---------------------------------+----------------+------------+-----------------+
 
 These intrinsics are available when ``arm_acle.h`` is included.
+
+Architectural Bridges
```

Comment:
nit: can we choose a common title for the top section on bridges? I vote for  `Architectural Extension Bridges` over `Architectural Bridges`.

```suggestion
Architectural Extension Bridges
```